terragrunt/aws/csp_violation_report_service/iam_lambda.tf

```diff
@@ -0,0 +1,29 @@
+data "aws_iam_policy_document" "purge_csp_reports_lambda_policies" {
+
+  statement {
+
+    effect = "Allow"
+
+    actions = [
+      "logs:CreateLogGroup",
+      "logs:CreateLogStream",
+      "logs:PutLogEvents"
+    ]
+
+    resources = [
+      "arn:aws:logs:ca-central-1:${var.account_id}:log-group:*"
+    ]
+  }
+
+  statement {
+
+    effect = "Allow"
+
+    actions = [
+      "lambda:InvokeFunction"
+    ]
+    resources = [
+      module.purge_csp_reports_lambda.function_arn
+    ]
+  }
+}
```

Comment:
Just noticed this - since this policy is only used by the Lambda function, does it need to the ability to invoke itself?  From what I could see in the code, the answer is no, but wanted to double check before removing.

Reply:
i think this is a copy paste fail from another project. Totally don't need this @patheard